install/kubernetes/helm/istio/values.yaml

```diff
@@ -325,6 +325,8 @@ global:
     #   cpu: 100m
     #   memory: 128Mi
 
+  # Specifies pod budget that reacts to disruptions with a PodDisruptionBudget(PDB).
+  # You can specify only one of maxUnavailable and minAvailable in a single PodDisruptionBudget.
```

Comment:
I was a bit confused for the comments and the code here:
1) You are updating the comments for `PodDisruptionBudget` but not adding `PodDisruptionBudget` support.
2) You are updating istio but not istio-remote.

Am I missing anything?

Reply:
The PR is to fix issue when user specify the `minAvailable` and `maxUnavailable` at the same time in single podDisruptionBudget definition.
And `istio-remote` is to be removed, see: https://github.com/istio/istio/pull/11307

Reply:
Done

Reply:
The title of this PR `add poddisruptionbudget to istio-remote.` should also be updated.